### PR TITLE
Modernize Qt connections

### PR DESCRIPTION
**Pull request by m-seker**
**Summary**

Get rid of old style Qt connections.

+

Some minor fixes.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Code style update
- [x] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of web configuration, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing setups:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's body (e.g. `Fixes: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated (docs/docs/en)
- [ ] Related tests have been updated

**PLEASE DON'T FORGET TO ADD YOUR CHANGES TO CHANGELOG.MD**
- [ ] Yes, CHANGELOG.md is also updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: include/hyperion/Hyperion.h [hunk 1]
```diff
@@ -398,8 +398,6 @@ public slots:
 	///
 	void currentImage(const Image<ColorRgb> & image);
 
-	void closing();
```
Comment (m-seker) [inline include/hyperion/Hyperion.h:401]: Nobody uses this

File: libsrc/hyperion/Hyperion.cpp [hunk 2]
```diff
@@ -151,16 +150,11 @@ void Hyperion::stop()
...
 
-void Hyperion::freeObjects(bool emitCloseSignal)
+void Hyperion::freeObjects()
 {
 	// switch off all leds
 	clear(-1,true);
 
-	if (emitCloseSignal)
-	{
-		emit closing();
```
Comment (m-seker) [inline libsrc/hyperion/Hyperion.cpp:161]: Nobody uses this

File: libsrc/leddevice/dev_other/LedDevicePiBlaster.cpp [hunk 3]
```diff
@@ -17,8 +17,6 @@ LedDevicePiBlaster::LedDevicePiBlaster(const QJsonObject &deviceConfig)
 
 	_activeDeviceType = deviceConfig["type"].toString("UNSPECIFIED").toLower();
 
-	signal(SIGPIPE,  SIG_IGN);
```
Comment (m-seker) [inline libsrc/leddevice/dev_other/LedDevicePiBlaster.cpp:20]: This is handled process wide from hyperiond

File: include/hyperion/Hyperion.h (hunk 1 above)
Comment (Lord-Grey) [reply include/hyperion/Hyperion.h:401]: Let's remove dead code....

File: libsrc/hyperion/Hyperion.cpp (hunk 2 above)
Comment (Lord-Grey) [reply libsrc/hyperion/Hyperion.cpp:161]: Let's remove dead code....

**Pull request merged**